p2p/src/channels/signaling/discovery/p2p_channels_signaling_discovery_reducer.rs

```diff
@@ -74,9 +82,10 @@ impl P2pChannelsSignalingDiscoveryState {
 
                 let dispatcher = state_context.into_dispatcher();
                 let message = SignalingDiscoveryChannelMsg::GetNext;
-                dispatcher.push(P2pChannelsSignalingDiscoveryEffectfulAction::MessageSend {
+                dispatcher.push(P2pChannelsEffectfulAction::SendMessage {
```

Comment:
Could we keep the same naming convention for actions? First nouns then verbs. `MessageSend`